### DISCUSSION TITLE
Dropdown icon

### DISCUSSION THREAD

**Discussion by irazetti** [twilio-labs/paste#2665] [General] [answered]
In AION, in one of the designs when the user clicks on a button icon it displays a dropdown. 

With Duncan (now he is gone) and one of the engineers, Ryan Kennedy, we were wondering if it wouldn't be more useful adding some type of indication (eg: chevron) that potentially could be clearer to the user that when clicking on the icon a dropdown menu will be displayed. 

**Please see image attached this could illustrate better the current behaviour:** 
![Group 281](https://user-images.githubusercontent.com/104209586/188845613-3c1c8814-127f-418d-bf80-123c3ebca2e0.jpg)
![Task header (1)](https://user-images.githubusercontent.com/104209586/188846383-2b5fa857-4bbc-40a0-b887-34b794323e7f.jpg)

Thanks!

**Comment by nkrantz** [comment, marked as answer]
Hi @irazetti! I'll post a more solid answer within the next few days after I chat with the team, but for now, are you using [the Menu component](https://paste.twilio.design/components/menu) for this dropdown? You might consider switching the MenuButton from an icon-only button to something a bit more descriptive, and including the down chevron icon.
<img width="206" alt="Screen Shot 2022-09-08 at 8 26 18 PM" src="https://user-images.githubusercontent.com/75342690/189248019-fc91609b-962a-4bb7-a6a3-584e18c2b977.png">

**Reply by irazetti** [reply]
Hi @nkrantz!
Thanks a lot for your answer!
I am wondering if we can also do something similar to this? Maybe we can already customise it with the current paste buttons.
![Group 288](https://user-images.githubusercontent.com/104209586/189320251-04af89d7-b410-4fdf-b99a-20a7340b927a.jpg)

**Reply by irazetti** [reply]
Not meaning to be the final one but to have it as a second option and see which one fits better for the whole AION product

**Reply by nkrantz** [reply]
@irazetti it seems that you have multiple options which is good. The best way for us to help you figure out which direction to take is by understanding your reasoning for making a change here. Is the current behavior getting complaints from customers? If you haven't already, it would be helpful to test out these different options with some users. If you haven't gotten any pushback about the existing implementation, it might not be worth making a change.

**Reply by irazetti** [reply]
@nkrantz thanks, yes, for the moment we won't make any change. 
Thanks!
